src/NUnitFramework/tests/Internal/Filters/NotFilterTests.cs

```diff
@@ -54,7 +54,7 @@ public void PassTest()
         {
             var filter = new NotFilter(new CategoryFilter("Dummy"));
 
-            Assert.False(filter.Pass(_topLevelSuite));
+            Assert.True(filter.Pass(_topLevelSuite));
             Assert.False(filter.Pass(_dummyFixture));
             Assert.False(filter.Pass(_dummyFixture.Tests[0]));
```

Comment:
There are no fixtures with two tests in them, as far as I can see. That's the exact case that was reported as not working.